dojo/db_migrations/0001_initial.py

```diff
@@ -488,6 +488,17 @@ class Migration(migrations.Migration):
                 'ordering': ['-date'],
             },
         ),
+        migrations.CreateModel(
```

Comment:
Hey @piyarathnalakmali, please do not modify existing migration files, this wrecks existing databases to do this. Instead, create a new one. Check [this](https://docs.djangoproject.com/en/2.2/topics/migrations/) out for more info. Thanks.